src/api/cpp/cvc5_kind.h

```diff
@@ -4071,10 +4063,29 @@ enum Kind : int32_t
    */
   INST_NO_PATTERN,
   /**
-   * Instantiation pool.
+   * Instantiation pool annotation.
+   *
+   * Specifies an annotation for pool based instantiation.  Should only be used
+   * as a child of INST_PATTERN_LIST.
    *
-   * Specifies an annotation for pool based instantiation.
+   * In detail, pool symbols can be declared via the method
+   *  - Solver::declarePool(const std::string&, const Sort&, const std::vector<Term>&) const
```

Comment:
I think we need an empty line after this for it to render correctly. Can you check?

Reply:
Yep, we do: 
https://cvc5.github.io/docs-ci/docs-pr8397/api/cpp/kind.html#_CPPv4N4cvc53api4Kind9INST_POOLE